src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al

```diff
@@ -226,6 +226,16 @@ table 149030 "AIT Test Suite"
                 end;
             end;
         }
+        field(60; "Imported by AppId"; Guid)
+        {
+            Caption = 'Imported from AppId';
+            ToolTip = 'Specifies the application id from which the test suite was created.';
+        }
+        field(70; "Imported XML's MD5"; Code[32])
```

Comment:
I do not think that we need this one on the Test Suite itself. It should probably go to the Test Input Group. 
Test Suite we have agreed not to touch for now when something new is published. Please let me know if I'm mistaking

Reply:
This is the pattern I have gone for:

1. Skip if the same suite is already imported by the same app
2. Error if the same suite is already imported with a different XML by the same app
3. Error if the same suite is already imported by a different app
4. Import any new suite

I think if we find issues while testing, we can revert this.

Reply:
Did not add MD5 check because of missing dependency on Sys App